src/options.cpp

```diff
@@ -2556,6 +2600,27 @@ string bool_to_string(bool val)
 }
 
 
+string tfi_to_string(TrueFalseIgnore_e val)
+{
+   switch (val)
+   {
+   case TFI_FALSE:
+      return("false");
+
+   case TFI_TRUE:
+      return("true");
+
+   case TFI_IGNORE:
+      return("ignore");
+
+   default:
+      fprintf(stderr, "tfi_to_string: Unknown argval '%d'\n", val);
```

Comment:
Shouldn't `__func__` be better here (and in every other log message) instead of the raw string?